test/rotoscope_test.rb

```diff
@@ -403,6 +410,35 @@ def test_dynamic_class_creation
     ], parse_and_normalize(contents)
   end
 
+  def test_dynamic_methods_in_blacklist
+    skip <<-FAILING_TEST_CASE
```

Comment:
Ah, indeed it does, because the assertion I'm doing is not the actual result I want to see 😄 I'll push up a failing one so it's more clear.